app/views/projects/show.slim

```diff
@@ -15,41 +13,7 @@ nav.page
     = link_to 'Destroy', project_path(@project), method: :delete, data: { confirm: 'This action cannot be undone. Are you sure?' }, class: 'edit btn btn-default btn-small'
   p
 
-table.table.table-striped.table-bordered
-  tr
-    th Mentor
-    td
-      = link_to @project.mentor_name, @project.mentor rescue @project.mentor_name
-      = " ("
-      = link_to @project.mentor_github_handle, "https://github.com/#{@project.mentor_github_handle}", class: 'github'
-      |)
-  tr
-    th Project Website
-    td = link_to @project.url, @project.url
-  tr
-    th Project Repository
-    td = link_to @project.source_url, @project.source_url
-  tr
-    th Suitable for Beginners?
-    td = @project.beginner_friendly? ? 'yes' : 'no'
-  tr
-    th Tags
-    td = project_tags @project
-  tr
-    th State
-    td = project_status(@project)
-  tr
-    th Applications (1st Choice)
-    td = @project.first_choice_application_drafts.count
-  tr
-    th Applications (2nd Choice)
-    td = @project.second_choice_application_drafts.count
-  tr
-    th Code of Conduct
-    td = link_to @project.code_of_conduct, @project.code_of_conduct
-  tr
-    th License
-    td = @project.license
+= render 'project_info', project: @project
```

Comment:
or shall we name the partial `_project.html.slim` and define it as the canonical way to display a project's details with its `to_partial_path`:

```
= render @project
```